qt/aqt/browser/sidebar/tree.py

```diff
@@ -932,6 +939,10 @@ def _maybe_add_type_specific_actions(self, menu: QMenu, item: SidebarItem) -> No
                     self.remove_tags_from_selected_notes,
                 )
 
+    def _maybe_add_add_action(self, menu: QMenu, item: SidebarItem) -> None:
+        if self._enable_add(item):
```

Comment:
I'd probably just inline the _enable_add/can_be_added_to checks here, so the code can be more easily followed.

Reply:
Check has been inlined in next commit